test/library_source_test.rb

```diff
@@ -91,6 +91,7 @@ def test_it_understand_local_absolute_git_paths
   end
 
   def test_it_understand_local_git_paths_with_backslashes
+    skip unless IS_WINDOWS
```

Comment:
These failed outside of Windows?

Reply:
Yes.  Why would one have a path with backslashes outside of Windows?

I think most people expect Ruby to translate from slash to backslash when using Windows, I not sure if it's needed anymore.  I don't think people expect the converse (backslash to slash)?

Reply:
I have to look closer at this. This didn't use to fail.